examples/wind/src/shaderlib/fs-fp32/fs-fp32.js

```diff
@@ -0,0 +1,159 @@
+// NOTE: this is same as luma.gl 'fp32' module except this is applied to fragment shader.
+const fsfp32shader = `\
```

Comment:
Maybe no need to copy the text? Can't we just import the luma.gl module and just create a new javascript object 
`export default {fs: vsmodule.vs}`
?